### PR TITLE
Add Hills Road as sponsor

### LINKED ISSUE #308: Add Hills Road Sixth Form College as sponsor

### PR DESCRIPTION
**Pull request by antoinepetty**
Fixes #308 
Needs confirmation from them that this is the desired url and name

### CODE REVIEW

File: images/sponsors/systemetric-hills-road.svg [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+       sodipodi:rx="1.5524604"
+       sodipodi:ry="1.3998399"
+       sodipodi:start="3.9634158"
+       sodipodi:end="3.8336157"
+       sodipodi:open="true"
+       sodipodi:arc-type="arc"
+       d="m 62.733557,23.116087 a 1.5524604,1.3998399 0 0 1 2.159161,0.03933 1.5524604,1.3998399 0 0 1 0.02645,1.947147 1.5524604,1.3998399 0 0 1 -2.157444,0.08702 1.5524604,1.3998399 0 0 1 -0.166448,-1.941501" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:48.9334px;line-height:1.25;font-family:'Niagara Engraved';-inkscape-font-specification:'Niagara Engraved';fill:#ffaaaa;fill-opacity:1;stroke:none;stroke-width:1.22333"
```
Comment (raccube) [inline images/sponsors/systemetric-hills-road.svg:359]: It seems this SVG expects a certain font to be available. This should really be converted to an outline instead.

Comment (antoinepetty) [reply images/sponsors/systemetric-hills-road.svg:359]: Are you able to do that or should I be asking them for it?

Comment (raccube) [reply images/sponsors/systemetric-hills-road.svg:359]: I'm able to do it. Turns out this is a font bundled with Windows.

Comment (raccube) [reply images/sponsors/systemetric-hills-road.svg:359]: In fact we actually need to do this, looks like this if you don't have the font.
image

**Comment by antoinepetty** [comment]
Thanks Karina :)

**Review by raccube — APPROVED** [review]

**Pull request merged**